tests/test_cinoh_Hamilton_Commission.py

```
@@ -0,0 +1,81 @@
+
+
+def test_description():
+    assert parsed_items[0]["description"] == ""
+
+
+def test_start():
+    assert parsed_items[0]["start"] == datetime(2025, 1, 7, 10, 0)
+
+
+def test_end():
+    assert parsed_items[0]["end"] == None
```

Ruff (0.8.2)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)